python/lambdas/ingest-cc-notifications/LICENCE

```diff
@@ -0,0 +1,18 @@
+Copyright (C) 2024 Crown Copyright (The National Archives)
```

Comment:
A general question, what happened the title in our license files? https://gds-way.digital.cabinet-office.gov.uk/manuals/licensing.html#use-mit

Reply:
I'm not sure what's happened here. There is a licence in the root of this project in the correct format though so we don't need this one.